namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -85,6 +85,7 @@ private[consul] object SvcAddr {
                 " Last known state is %s",
               datacenter, key.name, e, currentValueToLog
             )
+            state.update(Addr.Neg)
```

Comment:
there is a comment above on line 80 that is no longer accurate and needs to be updated